x-pack/plugins/apm/server/lib/service_map/transform_service_map_responses.ts

```diff
@@ -61,12 +63,35 @@ export function getServiceNodes(allNodes: ConnectionNode[]) {
   return serviceNodes;
 }
 
+function getServiceAnomalyData(
```

Comment:
Currently this returns

```ts
{
    [SERVICE_NAME]: string;
    maxAnomaly: MaxAnomaly;
}
```

I think it should simply return `MaxAnomaly` (which should be renamed to `ServiceAnomalyKPIs` or similar)